fbcp-ili9341.cpp

```diff
@@ -265,6 +307,15 @@ int main()
       memcpy(framebuffer[0], videoCoreFramebuffer[1], gpuFramebufferSizeBytes);
 #endif
 
+#ifdef LOW_BATT_PIN
+      if (tick() - lowBattLastPolled > LOW_BATT_POLLING_TIME) {
+        lowBatt = GET_GPIO(LOW_BATT_PIN) ? LOW_BATT_STATE : !LOW_BATT_STATE;
+        lowBattLastPolled = tick();
```

Comment:
Let's store `tick()` to a variable to only sample the register once:

```c++
uint64_t now = tick();
if (now - lowBatteryLastPolled > LOW_BATTERY_POLLING_TIME)
{
   lowBatteryLastPolled = now;
   ...
````

Reply:
I hesitated taking `now` in argument but there wasn't one value to pass in the call to pollLowBattery, so for the time being it's sampled in the method.